test/chainrules.jl

```diff
@@ -13,6 +13,15 @@
         end
     end
 
+    for x in -2.0:2.0
+        test_frule(xexpx, x)
+        test_rrule(xexpx, x)
```

Comment:
```suggestion
        test_scalar(xexpx, x)
```

Reply:
Didn't know about `test_scalar`. But that should be done for all the other rules too ...